sdk/typescript/test/bcs/bcs.test.ts

```diff
@@ -108,12 +107,122 @@ describe('Move BCS', () => {
         expect(data.kitties).toHaveLength(2);
         expect(data.owner).toEqual('00000000000000000000000000c0ffee');
     });
+
+    it('should de/ser TransactionData::Transfer', () => {
+        registerSuiCoreTypes(BCS);
```

Comment:
when writing multiple tests with a common setup step, i think it's helpful to abstract out that step into its own setup function. This helps the tests to stay easy to read and write

here, that would look like:
```typescript

describe('TransactionData', () => {
    beforeEach(() => {
        registerSuiCoreTypes(BCS);
    });
    
    it('should de/ser Self::Transfer', () => {
       // same test but with the register types line removed
    });
    
    // other tests with same change
});
```